internal/container/container.go

```diff
@@ -90,7 +95,7 @@ func (b *BootcCmd) Status(ctx context.Context) (*BootcHost, error) {
 	}
 
 	var bootcHost BootcHost
-	if err := yaml.Unmarshal([]byte(stdout), &bootcHost); err != nil {
+	if err := json.Unmarshal([]byte(stdout), &bootcHost); err != nil {
```

Comment:
We request the status in json - yaml unmarshal was actually _mostly_ working but failed for certain fields / cases